src/search/screens/search.screen.js

```diff
@@ -127,19 +161,22 @@ class Search extends Component {
     const selectedSearchType =
       selectedType !== null ? selectedType : this.state.searchType;
 
-    if (query !== '') {
+    if (query.trim() !== '') {
+      const searchedQuery = query.toLowerCase();
```

Comment:
GitHub search is case insensitive. We lowercase the query string in order to avoid having two paginations for nothing (Gitpoint vs gitpoint)